server/app/db/schemas/bots.py

```diff
@@ -0,0 +1,144 @@
+"""Contains all the schema for bots db layer,
+_"""
+
+from typing import Any, Dict, Tuple, Union
+
+from pydantic import BaseModel, Field, root_validator
+
+from .timestamp import TimestampInDBPlugin
+
+
+class BotBase(BaseModel):
+    name: str
+    bot_type: int
+
+
+class BotInDB(BotBase, TimestampInDBPlugin):
+    id: int
+
+    class Config:
+        orm_mode = True
+
+
+class BotInDBInflated(BotInDB):
+    """Bot db with bot_type data inflated"""
+
+    # bot_type_id: int
+    bot_type_name: str
+
+
+class CreateBot(BotBase):
+    pass
+
+
+class QueryBot(BaseModel):
+    name: str = Field("")
+    bot_type: Union[str, int] = Field(0)
+
+    _query_on = ""
+
+    def get_query_on(cls) -> str:
+        return cls._query_on
+
+    @root_validator
+    def check_valid_queried(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        "Checks if either of name or bot_type is given"
+        is_valid_name = cls.name != ""
+        is_valid_bot_type = False
+        if type(cls.bot_type is str):
+            if cls.bot_type == "":
+                is_valid_bot_type = False
```

Comment:
change these if else's to single line? `is_valid_bot_type = (cls.bot_type == "")`